src/components/pages/Home.vue

```diff
@@ -214,4 +228,16 @@ export default {
       color: darken($color-content-text-dark, 20%);
     }
   }
+
+  #code-button {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    border-radius: 5px;
+  }
+
+  #code-button:active {
+    transform: translateY(4px);
+  }
```

Comment:
```suggestion
    border-radius: 5px;

    &:active {
      transform: translateY(4px);
    }
  }
```
This can be shortened too